### PR TITLE
changed the command for create component

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
Signed-off-by: msivasubramaniaan <msivasub@redhat.com>

**Post Fix:**
getstarted-createComponent

### CODE REVIEW

**Comment by msivasubramaniaan** [comment]
> @msivasubramaniaan create from registry opens up quickpick, maybe we need to make it consistent and always open devfile UI to select the starter project.

Which devfile UI we can open? As user clicks create then we don't know which component he wants to create?

**Comment by mohitsuman** [comment]
I mean it says to Create a component using Registry Viewer, but it does not open the registry view webview.  So for now the heading should be `Create component from local workspace` and open the quick pick as we have.

**Comment by msivasubramaniaan** [comment]
> I mean it says to Create a component using Registry Viewer, but it does not open the registry view webview. So for now the heading should be `Create component from local workspace` and open the quick pick as we have.

Agree. Can you please provide me the Title and description, I can update the same

File: package.json [hunk 1]
```diff
@@ -1342,7 +1342,7 @@
 					{
 						"id": "createComponent",
 						"title": "Create a Component using Registry Viewer",
```
[the remark this answers is not in the document]
Comment (mohitsuman) [reply package.json:1344]: For line 1332 in package.json, replace this line `"title": "Browse all the Devfile stacks from Devfile registries",` with this: `"title": "Create component from Devfile registries webview",`

**Pull request merged**